src/components/widget/MasterWidget.js

```diff
@@ -30,50 +31,40 @@ class MasterWidget extends Component {
     updated: false,
     edited: false,
     data: '',
+    [this.props.fieldName]: this.props.widgetData,
```

Comment:
Can't we just use `field` or whatever ? Not that it's ever going to change throughout the lifetime of the widget.

Reply:
this.props.field is undefined ...  just  throw a console.log with it and see. I found the fieldName . to be most reliable.

Reply:
What I meant is just using `this.state.field` instead of using a dynamic name. I don't really see a reason for that, but also it's not that big of a deal.